src/main/java/com/minecraftabnormals/abnormals_core/core/util/DataUtil.java

```diff
@@ -106,8 +110,25 @@ public static boolean matchesKeys(ResourceLocation loc, RegistryKey<?>... keys)
 	 */
 	public static void registerAlternativeDispenseBehavior(Item item, BiPredicate<IBlockSource, ItemStack> condition, IDispenseItemBehavior newBehavior) {
 		IDispenseItemBehavior oldBehavior = DispenserBlock.DISPENSE_BEHAVIOR_REGISTRY.get(item);
-		DispenserBlock.registerDispenseBehavior(item, (source, stack) -> {
-			return condition.test(source, stack) ? newBehavior.dispense(source, stack) : oldBehavior.dispense(source, stack);
-		});
+		DispenserBlock.registerDispenseBehavior(item, (source, stack) -> condition.test(source, stack) ? newBehavior.dispense(source, stack) : oldBehavior.dispense(source, stack));
```

Comment:
I kind of liked the lamba return value being on its own line before, as it makes the code more readable.

Reply:
Yeah fair enough, I only changed it cause IntelliJ was whinging. Gotta figure out how to disable that inspection.

Reply:
It's reverted now.